qualtran/bloqs/factoring/ecc/ec_add_test.py

```diff
@@ -12,13 +12,456 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
+import pytest
+import sympy
+
 import qualtran.testing as qlt_testing
-from qualtran.bloqs.factoring.ecc.ec_add import _ec_add
+from qualtran._infra.data_types import QMontgomeryUInt
+from qualtran.bloqs.factoring.ecc.ec_add import (
+    _ec_add,
+    _ec_add_small,
+    _ECAddStepFive,
+    _ECAddStepFour,
+    _ECAddStepOne,
+    _ECAddStepSix,
+    _ECAddStepThree,
+    _ECAddStepTwo,
+    ECAdd,
+)
+from qualtran.resource_counting._bloq_counts import QECGatesCost
+from qualtran.resource_counting._costing import get_cost_value
+from qualtran.resource_counting.generalizers import ignore_alloc_free, ignore_split_join
+
+
+@pytest.mark.slow
+@pytest.mark.parametrize(
+    ['n', 'm'], [(n, m) for n in range(7, 10) for m in range(1, n + 1) if n % m == 0]
+)
+@pytest.mark.parametrize('a,b', [(15, 13), (2, 10), (8, 3), (0, 0)])
+@pytest.mark.parametrize('x,y', [(15, 13), (2, 10), (8, 3), (0, 0)])
+def test_ec_add_steps_classical_fast(n, m, a, b, x, y):
```

Comment:
this says fast but is slow and is marked as slow

Reply:
please consider taking a sampling of configurations rather than taking the "outer product" of point1 x point2 x n x m

Reply:
Marked correctly and reduced points to a few interesting cross products.